### PR TITLE
Downgrade errors in cert installer

### PR DESCRIPTION
**Pull request by nickbp**

### CODE REVIEW

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java [hunk 1]
```diff
@@ -33,14 +33,14 @@ protected static boolean installCertificate(String pathToJRE, ProcessRunner proc
...
-                LOGGER.error("JRE not found at path: {}", jrePath.toAbsolutePath());
+                LOGGER.warn("JRE not found at path: {}", jrePath.toAbsolutePath());
                 return false;
             }
 
             final Path sandboxCertPath = Paths.get(CERT_PATH);
             // Check if cert is present, there's nothing to do.
             if (!Files.exists(sandboxCertPath)) {
-                LOGGER.error("Cert file not found at path: {}", sandboxCertPath.toAbsolutePath());
+                LOGGER.info("Cert file not found at path, this is expected on most clusters: {}", sandboxCertPath.toAbsolutePath());
```
Comment (vespian) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java]: `this is expected on most clusters` -> `this is expected on non-strict EE or Open clusters` ?

as Dan, I do not know what qualifies my cluster as the one belonging to group `most`

**Comment by nickbp** [comment]
Clusters do not tell us these things.

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java (hunk 1 above)
Comment (nickbp) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java]: Done. Went with "Cert file not found at the expected path, nothing to do. This is expected when the cluster is not in strict mode"

**Comment by nickbp** [comment]
retest this please (jenkins restarted? never actually started this build)

**Comment by mohitsoni** [comment]
@mesosphere-ci retest this please

**Pull request merged**